apps/frontend/src/app/pages/company/create-profile/create-profile.component.html

```diff
@@ -0,0 +1,366 @@
+<form class="container px-5 mx-auto min-h-screen" [formGroup]="form">
+  <div class="md:flex gap-36 pt-8">
+    @if(step === 1){
+    <div class="w-full">
+      <div class="md:flex w-full gap-8">
+        <div
+          class="container flex justify-end bg-neutral border-2 rounded-3xl border-base-300 w-48 h-48"
+        >
+          @if(!profilePhoto){
+          <div class="rounded-3xl bg-neutral object-cover w-48 h-46"></div>
+          } @if(profilePhoto){
+          <img
+            class="rounded-3xl object-cover w-48 h-46"
+            [src]="profilePhoto"
+            alt="profile company"
+          />
+          }
+          <div
+            class="absolute rounded-3xl pt-40 pr-3 w-9 h-9 flex items-center justify-center cursor-pointer"
+            (click)="profilePhoto.click()"
+          >
+            <input
+              #profilePhoto
+              class="hidden"
+              type="file"
+              accept="image/*"
+              (change)="onFileChange($event)"
+            />
+            <i
+              class="bi bi-pencil text-sm text-primary shadow bg-neutral py-2 px-3 rounded-full"
+              aria-hidden="true"
+            ></i>
+          </div>
+        </div>
+        <div class="w-full">
+          <div class="w-full">
+            <wagademy-form-field
+              [control]="form.controls.companyName"
+              label="Company name"
+            >
+              <wagademy-input placeholder="Company name"></wagademy-input>
+            </wagademy-form-field>
+            <div class="pt-4">
+              <wagademy-form-field
+                [control]="form.controls.areaOfExpertise"
+                label="Area of expertise"
+              >
+                <wagademy-input
+                  placeholder="Area of expertise"
+                ></wagademy-input>
+              </wagademy-form-field>
+            </div>
+            <div class="pb-6">
+              <wagademy-form-field
+                [control]="form.controls.description"
+                label="Description"
+              >
+                <textarea
+                  class="h-28 w-full outline-none p-2 text-base-100 bg-neutral"
+                  placeholder="Tell us more about yourself"
+                ></textarea>
+              </wagademy-form-field>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="flex md:justify-end justify-center md:pt-36">
+        <button class="btn btn-primary w-[200px]" (click)="step = 2">
+          Next
+        </button>
+      </div>
+    </div>
+    } @if(step === 2){
```

Comment:
Here also the responsiveness is not working well and the screen is not equal figma:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/e0f8203d-cd4f-4e9c-96eb-30afb4869ab3)
How it is in figma:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/63c5a871-eef2-4b74-9ff5-9d2c5e3a682f)

Also for cell phone is missing and spacing between the buttons and the boxes above:

![image](https://github.com/Venture-Miner/wagademy/assets/90660988/2247350f-a0ab-4b00-a3fe-4303909f2031)